### PR TITLE
update performance evaluation form to require comment if answer is NA

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Form - require comment if answer is NA#170](https://github.com/nerubia/kh360-react/issues/170)

BE PR: [add get email templates for NA message and save NA rating comment #160](https://github.com/nerubia/kh360-nodejs/pull/160)

### CODE REVIEW

File: src/components/rating/StarRating.tsx
Comment (jatnerubia) [inline src/components/rating/StarRating.tsx:1]: refactor the file name and move to shared folder 🤔

File: src/features/evaluations/[id]/EvaluationsCriteria.tsx
Comment (jatnerubia) [inline src/features/evaluations/[id]/EvaluationsCriteria.tsx:1]: refactor the file name

File: src/types/evaluationRatingType.ts
Comment (jatnerubia) [inline src/types/evaluationRatingType.ts:1]: refactor the file name

File: src/types/evaluationTemplateContentType.ts
Comment (jatnerubia) [inline src/types/evaluationTemplateContentType.ts:1]: refactor the file name

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/components/rating/star-rating.tsx
Comment (jatnerubia) [inline src/components/rating/star-rating.tsx:1]: move to `ui` or `shared` folder 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**